CMakeLists.txt

```diff
@@ -382,6 +382,8 @@ if(CORENRN_ENABLE_CALIPER_PROFILING)
   include_directories(${caliper_INCLUDE_DIR})
   add_definitions("-DCORENEURON_CALIPER")
   set(CALIPER_LIB "caliper")
+  set_property(GLOBAL APPEND_STRING PROPERTY CORENEURON_LIB_LINK_FLAGS
+                                             " -L${caliper_LIB_DIR} -l${CALIPER_LIB}")
```

Comment:
Curious why this is needed here? Because, from my earlier testing, `nrnivmodl-core` should be already working when calliper is enabled.

Reply:
As discussed offline this is for enabling caliper when running `nrnivmodl -coreneuron` when `CoreNEURON` is installed as submodule of `NEURON`. `CORENEURON_LIB_LINK_FLAGS` variable is passed to `nrnmech_makefile` and includes the needed flags to link caliper to the created libraries.